formfactory/tests/test_models.py

```diff
@@ -65,10 +67,21 @@ def test_fieldgroup(self):
             self.assertEqual(getattr(self.fieldgroup, key), value)
 
     def test_wizard(self):
-        self.assertQuerysetEqual(
-            self.wizard.forms.all().order_by("wizardformthrough"),
-            [repr(self.simpleform), repr(self.loginform)]
-        )
+        import django
+        if django.get_version() == "1.9":
```

Comment:
Drop a comment here please. Why 1.9 specific code?

Reply:
Will add. issue was introduced in 1.9 from 1.8x. And will make 110% but was fixed in 1.9.1.

Reply:
It was fixed as of 1.9.2